common/src/force.rs

```diff
@@ -0,0 +1,64 @@
+use serde::{Deserialize, Serialize};
+/* GravityMethod encodes different ways of creating gravity in Hyperbolic space
+
+PlanarConstant has gravity oriented in the "down" direction, and the force of gravity is
+constant with repect to height.
+
+PlanarConstant has gravity oriented in the "down" direction, and the force of gravity respects
+the divergence theorem, which makes it proportial to the ratio of the area of the ground-plane
+to the area of the surface which is equidistant to the ground-plane at the indicated height.
+
+Zero disables gravity.
+*/
+#[derive(Debug, Deserialize, Serialize, Copy, Clone)]
+pub enum GravityMethod {
+    PlanarConstant,
+    PlanarDivergent,
+    Zero,
+}
+
+pub struct ForceParams {
+    pub gravity_intensity: f64,
+    pub air_drag_factor: f64, // lower number means more drag
+    pub gravity_type: GravityMethod,
+    pub float_speed: f64,
+}
+
+impl ForceParams {
+    pub fn apply_forces(
```

Comment:
`/// returns the result of applying appropriate forces to "inital_velocity" over "time"`

Reply:
Since you hadn't marked this as resolved, I assume you would like my input. My main suggestion would be to revise the wording to `returns the resulting velocity after applying ...` to make it clearer that a velocity is returned. It may also be useful to rename the method to something like `get_velocity_from_forces`.